### DISCUSSION TITLE
How do I prevent beachball from changing the workspace versions in devDependencies?

### DISCUSSION THREAD

**Discussion by ilancohen** [microsoft/beachball#940] [Q&A]
I have a monorepo set up with pnpm with workspaces and beachball.
For packages with peer dependencies, I add the same package in both `peerDependencies` and `devDependencies`, like so:

```
{
   name: "package-with-deps"
   devDependencies: {
       local-dep: "workspace:^"
   },
   peerDependencies: {
       local-dep: "^3.6.1"
   }
}
```

but when I release a new version of `local-dep`, beachball replaces the `workspace:^` with a numeric version. Why?

**Comment by ecraig12345** [comment]
Cleaning up old discussions--this seems to have been fixed at some point
